analyticConfigurations/metric_meta-aws.lambda.json

```diff
@@ -28,6 +28,23 @@
           },
           "validMax": 100
         }
+      },
+      {
+        "match": "aws.lambda.memory.utilization",
+        "properties": {
+          "tags": {
+            "unit": "percent"
+          },
+          "validMax": 100
+        }
+      },
+      {
+        "match": "aws.lambda.billed",
+        "properties": {
+          "tags": {
+            "unit": "GB-seconds"
```

Comment:
Based on KBN documentation (https://github.com/Netuitive/kbn/blob/e84bd776cce512cb6c9569885e7f49eb0de16338/README.md#L76), the gigabytes / second unit should be referenced as `gBps`.

Reply:
Wow, nice find. I honestly didn't think it'd be a unit in KDB. My bad.